etc/kayobe/ansible/scripts/smartmon.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+import subprocess
+import json
+from datetime import datetime
+
+SMARTCTL_PATH = "/usr/sbin/smartctl"
+
+def run_command(command, parse_json=False):
+    result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
+    if parse_json:
+        return json.loads(result.stdout)
+    else:
+        return result.stdout.strip()
+
+def parse_smartctl_attributes(disk, disk_type, serial, json_data):
+    labels = f'disk="{disk}",type="{disk_type}",serial_number="{serial}"'
+    metrics = []
+    smartmon_attrs = set([
```

Comment:
style nit: It would be helpful to move these to a 'constant' eg. `SMARTMON_ATTRS = set(..` and put each attribute on a new line.